crates/bevy_ui/src/ui_node.rs

```diff
@@ -230,10 +307,13 @@ impl Default for Overflow {
     }
 }
 
+/// The type of position of the node
 #[derive(Copy, Clone, PartialEq, Debug, Serialize, Deserialize, Reflect)]
 #[reflect_value(PartialEq, Serialize, Deserialize)]
 pub enum PositionType {
+    /// Relative to all other nodes with the [`PositionType::Relative`] value
     Relative,
+    /// Independent of all other nodes
```

Comment:
I think it's only independent from siblings, but still relative to it's parent

Reply:
PositionType::Relative actually makes it influenced by seemingly unrelated other nodes if they also have PositionType::Relative, but I've never actually set absolute on a node with a parent, so I'm not sure what happens when you do that 🤔

Reply:
IIUC, with PositionType::Absolute, it stops participating in the layout of its parent and siblings, and sets it's local transform and node size according to style.position.

Reply:
It also stops being influenced by unrelated nodes with relative positioning, I was struggling with layouting because stuff was unexpectedly interacting with each other without being related

Reply:
> stuff was unexpectedly interacting with each other without being related

Could you give more details about that?
To be clear, this is what I meant:
- if style.position_type == PositionType::Relative, the node takes part in it's parent/siblings' flexbox layout, which sets it's _default_ position (and "interacts" with their _default_ position)
- if style.position_type == PositionType::Absolute, the node does not take part in it's parent/siblings' layout, and it's _default_ position is the position of it's parent (that is, it's local transform is 0)
- style.position displaces the node from its default position after the layout algo is finished. it doesn't "push" other nodes, regardless of the node's position_type.

Reply:
What I mean is that nodes with a relative positioning also influence other nodes with relative positioning that aren't in the same hierarchy at all - this is a counterintuitive issue I had to wrangle for one of my games

Reply:
That's because there is effectively only one layout hierarchy. All the "root" UI nodes are children of a "window" node (which only exists during layout).